plugins/nf-wave/src/main/io/seqera/wave/plugin/config/WaveConfig.groovy

```diff
@@ -71,7 +73,7 @@ class WaveConfig {
 
     protected List<String> parseStrategy(value) {
         if( !value ) {
-            final defaultStrategy = List.of('container','dockerfile','conda')
+            final defaultStrategy = List.of('container','dockerfile','conda', 'spack')
```

Comment:
It'd be better to turn this into a constant and use it also in the check below (line 88)

Reply:
done!